### PR TITLE
Fix test for travis

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

**Comment by frankrousseau** [comment]
Travis is still red....

**Comment by aenario** [comment]
Hopefully not for long

File: server/controllers/account.coffee [hunk 1]
```diff
@@ -68,7 +68,8 @@ module.exports =
                     return cb null, errors
 
                 unless utils.checkPassword oldPassword, user.password
-                    errors.push localizationManager.t "current password incorrect"
+                    errors.push localizationManager.t "current password" +
+                                                                " incorrect"
```
Comment (nono) [inline server/controllers/account.coffee]: @aenario I'd prefer not to split translation keys. This is something we may want to grep to see if it is still used when updating the locale files. Maybe something like:

```coffee
msg = localizationManager.t "current password incorrect"
errors.push msg
```

**Comment by aenario** [comment]
Fixed, you're right, thanks for the review !

**Pull request merged**